app/policies/work_version_policy.rb

```diff
@@ -28,19 +28,21 @@ def create?
   end
 
   # Can edit a work iff:
-  #   The work is in a state where it can be updated (e.g. not depositing)
+  #   The work is in a state where it can be updated (e.g. not depositing, not an in-progress purl reservation)
   #   AND if any one of the following is true:
   #     1. The user is an administrator
   #     2. The user is the depositor of the work and it is not currently pending approval (review workflow)
   #     3. The user is a manager of the collection the work is in
   #     4. The user is a reviewer of the collection the work is in
   sig { returns(T::Boolean) }
   def update?
+    return false unless record.updatable?
```

Comment:
This would break our existing requirement that admins can edit things that are in an in-review state.

Reply:
for posterity:  discussed on slack w/ @jcoyne.  Determined that this would still allow admins to edit things that are in-review (since the state machine allows `update_metadata!` on `pending_approval` work versions).  This _does_ break the assumption that admins can edit anything at any time, by disallowing them from editing things in the `depositing` state, but I confirmed with @amyehodge that this was acceptable (like `purl_requested` and `reserving_purl`, `depositing` should be a pretty fleeting state when all is working normally).